README.md

```diff
@@ -29,21 +29,27 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
+    The DOM is the browser's interpretation of your code.  You can manipulate the interpretation with Javascript to enhance your site without having to change the HTML or CSS directly.
 
 2. What is an event?
+    An event is when some type of interaction occurs, like a mouse click or key press.
 
 3. What is an event listener?
+    An event listener specifies what type of event to be on the lookout for, and what to do when the event happens.
 
 4. Why would we convert a NodeList into an Array?
+    You would convert the NodeList to allow access to array methods.
 
 5. What is a component?
+    A component is an inclusive segment of code, usually classes and methods that is reusable and creates a specific component for your website, like carousels and cards.  
```

Comment:
great answers, this will help if you look back and forget anything about dom and event listeners